test.js

```diff
@@ -1,148 +1,162 @@
-/* jshint mocha:true */
+/* eslint-env mocha */
 'use strict';
 
-var assert = require('assert');
-var Plugin = require('./');
+require('chai').should();
+const Plugin = require('./');
 
 describe('Plugin', function() {
-  var plugin;
+  let plugin;
   this.timeout(10000);
 
-  beforeEach(function() {
-    plugin = new Plugin({ paths: { root: '.' }});
+  beforeEach(() => {
+    plugin = new Plugin({paths: {root: '.'}});
   });
 
-  it('should be an object', function() {
-    assert(plugin);
-  });
+  it('should have #compile method', () =>
```

Comment:
Does Mocha care about what is returned?

Anyway, yeah, that's not a good practice. Gonna add curly brackets around it.